Spika/UIFlow/MainFlow/Chats/CurrentChat/View/CurrentChatView.swift

```diff
@@ -111,7 +111,7 @@ class CurrentChatView: UIView, BaseView {
     }
     
     func moveInputFromBottom(for n: CGFloat) {
```

Comment:
What does the 30 here represent? Is this hardcoded approximation?

Reply:
i dont see 30?